.drone.yml

```diff
@@ -51,21 +42,19 @@ publish:
     password: $$DOCKER_PASS
     email: $$DOCKER_EMAIL
     repo: hypriot/rpi-swarm
+    file: Dockerfile
     tag:
-      - 1.2.3
-      - latest
-    save:
-      file: /drone/src/buildresult/latest/swarm.tar
+      - $$TAG
+    when:
```

Comment:
why not tagging latest as well?
A new tag build should update latest as well on Docker Hub. WDYT?